chart.js/index.d.ts

```diff
@@ -3,15 +3,15 @@
 // Definitions by: Alberto Nuti <https://github.com/anuti>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare enum ChartType {
+export enum ChartType {
```

Comment:
I am changing it to `declare type ChartType = 'line' | 'bar' | ...`.
The problem is that chart.js can be extended by additional ChartTypes. Therefore in ChartConfiguration I define `type?: ChartType | string;` leaving space for future enhancements.